mllib/src/test/java/org/apache/spark/ml/classification/JavaRandomForestClassifierSuite.java

```diff
@@ -80,6 +80,12 @@ public void runDT() {
     for (String featureSubsetStrategy: RandomForestClassifier.supportedFeatureSubsetStrategies()) {
       rf.setFeatureSubsetStrategy(featureSubsetStrategy);
     }
+    for (double featureSubsetFraction = 0.1; featureSubsetFraction <= 1.0; featureSubsetFraction += 0.1) {
```

Comment:
I'm not sure there is a good reason to try out this many values. Perhaps you can just add a couple different string test cases to the for loop above. I don't see a reason to test every integer from 1 to 100, just a couple edge cases should work.